api/python/quilt3/packages.py

```diff
@@ -225,6 +226,41 @@ def get_cached_path(self):
             return ObjectPathCache.get(physical_key)
         return None
 
+    def get_bytes(self, use_cache_if_available=True):
+        """
+        Returns the bytes of the object this entry corresponds to. If 'use_cache_if_available'=True, will first try to
+        retrieve the bytes from cache.
+        """
+        if use_cache_if_available:
+            cached_path = self.get_cached_path()
+            if cached_path is not None:
+                cached_file_url = fix_url(cached_path)
```

Comment:
`fix_url` is intended strictly for user input, where we want to handle both URLs and file paths (and rely on heuristics - e.g.,you could technically have a local file named `s3:`, but it's unlikely). Here, you know it's going to be a file path, so just use `pathlib`:
`pathlib.Path(cached_path).as_uri()`
or just read the bytes directly:
`pathlib.Path(cached_path).read_bytes()`